src/utility.ts

```diff
@@ -423,6 +426,16 @@ export class Utility {
         await Constants.ExtensionContext.globalState.update(Constants.StateKeyIoTHubID, "");
     }
 
+    // Temp utility to solve the compatible issue because of the schema change in IoT Hub Service.
+    // moduleContent -> modulesContent
+    public static updateSchema(deployment: any): any {

```

Comment:
Where is this used? Do we confirmed about the schema change with IoT Edge team?

Reply:
Deleted.